tests/php/Authenticator/RegisterHandlerTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace SilverStripe\MFA\Tests\Authenticator;
+
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\FunctionalTest;
+use SilverStripe\MFA\Authenticator\MemberAuthenticator;
+use SilverStripe\MFA\Extension\MemberExtension;
+use SilverStripe\MFA\Store\SessionStore;
+use SilverStripe\MFA\Tests\Stub\BasicMath\Method;
+use SilverStripe\Security\Member;
+
+/**
+ * Class RegisterHandlerTest
+ *
+ * @package SilverStripe\MFA\Tests\Authenticator
+ */
+class RegisterHandlerTest extends FunctionalTest
+{
+    const URL = 'Security/login/default/mfa/register/SilverStripe-MFA-Tests-Stub-BasicMath-Method/';
+
+    protected static $fixture_file = 'RegisterHandlerTest.yml';
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        Injector::inst()->load([
+            'SilverStripe\Security\Security' => [
```

Comment:
```suggestion
            Security::class => [
```

(Also need to import the namespace)